pyGPG/gpg.py

```diff
@@ -122,6 +169,20 @@ def listkey(self, id_string=None, outputfile=None):
         return self.runGPG('list-key', inputfile=id_string,
                            outputfile=outputfile)
 
+<<<<<<< HEAD
+=======
+    def importkey(self, inputfile=None):
+        '''Import keys with --import <inputfile>
+
+        @param inputfile: string filepath to pass to
+                           gpg for the key to import
+        @rtype GnuPGResult object
+        '''
+        return self.runGPG('import', inputfile=inputfile,
+                           #outputfile=outputfile
+        )
+
```

Comment:
I'm not 100% sure we need this convenience function.  I did want to keep the main library quite lightweight.  If it was expanded to handle the various forms of import maybe...  In the user app we are working on using this lib, It uses runGPG directly.